guides/common/modules/proc_importing-a-subscription-manifest-into-foreman-server.adoc

```diff
@@ -7,6 +7,11 @@ ifdef::foreman-el,katello[]
 This is for users of the Katello plug-in and Red Hat operating systems only.
 endif::[]
 
+[NOTE]
+====
+If Simple Content Access (SCA) is enabled on your organization and you import a non-SCA manifest, it will show as SCA.
```

Comment:
I am not sure I understand this. Is this relevant for the user?

Reply:
Here's a better way to put it:

```suggestion
Simple Content Access (SCA) is now set on the organization, not the manifest. Importing a manifest will not change your organization's Simple Content Access status.
```

Reply:
Thanks! looks like I didn't quite catch the meaning first time.